pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -29,6 +29,25 @@ def test_freq_validation_with_nat(self, dt_cls):
         with pytest.raises(ValueError, match=msg):
             dt_cls([pd.NaT, pd.Timestamp("2011-01-01").value], freq="D")
 
+    # TODO: better place for tests shared by DTI/TDI?
+    @pytest.mark.parametrize(
+        "index",
+        [
+            pd.date_range("2016-01-01", periods=5, tz="US/Pacific"),
+            pd.timedelta_range("1 Day", periods=5),
+        ],
+    )
+    def test_shallow_copy_inherits_array_freq(self, index):
```

Comment:
note that we appear to have very limited shallow_copy tests now :-> so this is good